File: ts/iq/tos~set~trackable.ts
/** <iq type="set" xmlns="tos">. */
export function buildUserNoticeSetResult(id: number, result: number): BinaryNode {
  return { tag: 'iq', attrs: { to: 's.whatsapp.net', xmlns: 'tos', type: 'set' }, content: [
      { tag: 'trackable', attrs: { id: id, result: result } },
    ] }
}

File: rust/iq/tos~set~trackable.rs
/// <iq type="set" xmlns="tos">.
pub fn build_user_notice_set_result(id: i64, result: i64) -> Node {
    NodeBuilder::new("iq")
        .attr("to", "s.whatsapp.net")
        .attr("xmlns", "tos")
        .attr("type", "set")
        .child(NodeBuilder::new("trackable").attr("id", id).attr("result", result).build())
        .build()
}